### DISCUSSION TITLE
Issue connecting to MQTT broker with SSL enabled

### DISCUSSION THREAD

**Discussion by nikhilshetty1596** [emqx/MQTTX#1242] [Q&A]
Hi folks, anyone who has tried connecting to MQTT broker which is SSL enabled? I am looking for help on how to generate the following certificates:
CA File
Client Certificate File
Client key file
Any help would be appreciated:)

**Comment by ysfscream** [comment]
I hope these two posts can help you.

1. https://www.emqx.com/en/blog/emqx-server-ssl-tls-secure-connection-configuration-guide

2. https://www.emqx.com/en/blog/enable-two-way-ssl-for-emqx#enable-and-verify-ssl-tls-two-way-connection
